clean.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+echo "Cleaning repo"
+echo "Removing __pycache__/ ..."
+rm -rf __pycache__/
+
+echo "Removing mobile_testkit_tests/__pycache__/ and .cache ..."
+rm -rf mobile_testkit_tests/__pycache__/
+rm -rf mobile_testkit_tests/.cache/
```

Comment:
Fixed. We may need to update this as well discover more pycaches. I imagine that the other suites will generate them as well.